connectors/sources/confluence.py

```diff
@@ -61,6 +64,31 @@
 CONFLUENCE_SERVER = "confluence_server"
 WILDCARD = "*"
 
+ACCESS_CONTROL = "_allow_access_control"
+
+
+def _prefix_identity(prefix, identity):
```

Comment:
I've noted in the Jira PR, that we should move these functions to a common `access_control.py` module, but that'll be on my side 👍